modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/Diagram/Diagram.tsx

```diff
@@ -156,6 +156,15 @@ function DiagramBuilder() {
 		}
 	};
 
+	const setHandleConnection = (isConnectable: boolean) => {
```

Comment:
i think it would be better to rename `setHandleConnection` to `setNodeHandleConnection`, what do you think?